index.html

```diff
@@ -51,33 +58,185 @@ <h1 class="about-title padding">I'm Rose Glad to see you!</h1>
+          <li class="career-role">Back End Dev</li>
+          <li class="small-circle">.</li>
           <li class="year">2015</li>
         </ul>
         <div class="project-description">
           <p>A daily selection of privately personalized
             reads; no accounts or sign-ups required.</p>
         </div>
-        <ul class="skills">
-          <li><a href="#">html</a></li>
-          <li><a href="#">css</a></li>
-          <li><a href="#">javascript</a></li>
+        <ul class="my-skills">
+          <li><a href="#" class="skill-link html-btn">html</a></li>
+          <li><a href="#" class="skill-link css-btn">css</a></li>
+          <li><a href="#" class="skill-link js-btn">javascript</a></li>
         </ul>
-        
-        <a href="#">See Project</a>
+      
+        <a href="#" class="see-project-btn">See Project</a>
       </div>
+
+      <div class="card project4">
+        <div class="project-image">
+          <img src="./images/SnapshootPortfolio3.svg" alt="" srcset="">
+        </div>
+        <h3 class="project-title">Multi-Post Stories</h3>
+        <ul class="project-links">
+          <li class="canopy">CANOPY</li>
+          <li class="small-circle">.</li>
+          <li class="career-role">Back End Dev</li>
+          <li class="small-circle">.</li>
+          <li class="year">2015</li>
+        </ul>
+        <div class="project-description">
+          <p>A daily selection of privately personalized
+            reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="my-skills">
+          <li><a href="#" class="skill-link html-btn">html</a></li>
+          <li><a href="#" class="skill-link css-btn">css</a></li>
+          <li><a href="#" class="skill-link js-btn">javascript</a></li>
+        </ul>
       
+        <a href="#" class="see-project-btn">See Project</a>
+      </div>
+
+      <section class="about-rose">
+        <div class="about-title-section">
+          <h1 class="about-title padding">About Myself</h1>
+        </div>
+        <div class="full-description">
+          <p class="about-me padding">Hello I’m a software developer! 
+            I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="socials-section">
+          <button class="connect-link padding">LET'S CONNECT</button>
+          <ul class="icons">
+            <li><img src="./images/twitter.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/Linkedin.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/mail.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/github.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/playboy.svg" class="icon" alt="twitter icon"></li>
+          </ul>
```

Comment:
Please consider adding a `a tag` wrapping each `img tag`.

## Reasons:-

1- It will make it semantically correct.
2- Users can reach you by using your social media.

## What to do now:-

You can replace above lines of code with the following code:-

![image](https://user-images.githubusercontent.com/94223367/219571099-00930904-3b5b-4489-aeb6-f0cc259a95ee.png)

Or use the following code:-

`    <ul class="icons">
      <li>
        <a href="YOUR SOCIAL MEDIA HANDLE">
          <img src="./images/twitter.svg" class="icon" alt="twitter icon" />
        </a>
      </li>
      <li>
        <a href="YOUR SOCIAL MEDIA HANDLE">
          <img src="./images/Linkedin.svg" class="icon" alt="twitter icon" />
        </a>
      </li>
      <li>
        <a href="YOUR SOCIAL MEDIA HANDLE">
          <img src="./images/mail.svg" class="icon" alt="twitter icon" />
        </a>
      </li>
      <li>
        <a href="YOUR SOCIAL MEDIA HANDLE">
          <img src="./images/github.svg" class="icon" alt="twitter icon" />
        </a>
      </li>
      <li>
        <a href="YOUR SOCIAL MEDIA HANDLE">
          <img src="./images/playboy.svg" class="icon" alt="twitter icon" />
        </a>
      </li>
    </ul>`